### PR TITLE
Lesson 04 Part 03: Make

### PR DESCRIPTION
**Pull request by Ayrum1158**
Have compilation issue, error screenshot attached.

Screenshot from 2019-03-30 16-47-13

### CODE REVIEW

**Comment by Ayrum1158** [comment]
Done Lesson 04 Part 03: Make file implementation

@a-vodka @marynasergiyenko

File: 01-Git_Bash_Make/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FUNC=guessanumber
+MAIN=main
+HEAD=header
+
```
Comment (a-vodka) [inline 01-Git_Bash_Make/Makefile:5]: please define gcc as variable

**Comment by a-vodka** [comment]
> Have compilation issue, error screenshot attached.
> 
> Screenshot from 2019-03-30 16-47-13

To solve this issue you have to prevent multiple including. Please read following link
http://qaru.site/questions/1090720/what-is-a-guard-block-for-a-header-file-in-c

**Comment by Ayrum1158** [comment]
Done lesson 4 part 3: Makefile implementation

**Pull request merged**